impl/src/main/java/org/ehcache/impl/internal/store/tiering/CompoundCachingTier.java

```diff
@@ -67,6 +68,9 @@ public void onInvalidation(final K key, final Store.ValueHolder<V> valueHolder)
         }
       }
     });
+
+    StatisticsManager.associate(higher).withParent(this);
+    StatisticsManager.associate(lower).withParent(this);
```

Comment:
Calling static method through a subclass reference... might be better to call ContextManager.associate(...)